app/json/mockRoutes.json

```diff
@@ -0,0 +1,7 @@
+{"rows":
```

Comment:
I was using it before the REST calls were implemented and I forgot to remove it.